lib/steps.js

```diff
@@ -812,6 +812,11 @@ When(/I answer randomly for at most ([0-9]+) (?:page(?:s)?|screen(?:s)?)/, { tim
         if ( has_thrown ) { return; }
         else { has_thrown = true; }
 
+        if (!( await scope.is_on_interview_still(scope))) {
+          await scope.page.goBack();
+          resolve();
+        }
```

Comment:
Should this be in a loop? Not sure that's really useful. Also, should it use `networkIdle` or whatever?